sound/soc/intel/boards/sof_sdw.c

```diff
@@ -178,6 +178,17 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_SDW_PCH_DMIC |
 					RT711_JD2),
 	},
+	{
+		/* NUC15 'Bishop County' LAPBC510 and LAPBC710 skews */
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Intel(R) Client Systems"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "LAPBC"),
```

Comment:
Hmm. First I thought this is a bit broad pattern, but it seems all the LAPBC skews share the design, so seems safe.

Reply:
yes, I checked the pattern and the 'LAP' identifies a product type (LAPtop) and 'BC' is the product name, so no need to check the rest.